.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -7,6 +7,8 @@
 *Checklist:*
 
 - [ ] Write/update tests
```

Comment:
We could go with sub lists:
- [ ] Add/update tests using:
    - [ ] Correct values
    - [ ] Bad/wrong values (wrong type, length, etc.)
    - [ ] [Intrinsic Functions](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/intrinsic-function-reference.html)

Reply:
Great suggestion. Much easier to read and understand.